iteration/iteration.go

```diff
@@ -86,3 +94,23 @@ func (m *GormIterationRepository) Load(ctx context.Context, id uuid.UUID) (*Iter
 	}
 	return &obj, nil
 }
+
+// Save updates the given iteration in the db. Version must be the same as the one in the stored version
+// returns NotFoundError, VersionConflictError or InternalError
+func (m *GormIterationRepository) Save(ctx context.Context, i Iteration) (*Iteration, error) {
+	itr := Iteration{}
+	tx := m.db.Where("id=?", i.ID).First(&itr)
+	if tx.RecordNotFound() {
+		// treating this as a not found error: the fact that we're using number internal is implementation detail
+		return nil, errors.NewNotFoundError("iteration", i.ID.String())
+	}
+	if err := tx.Error; err != nil {
+		return nil, errors.NewInternalError(err.Error())
+	}
+	tx = tx.Save(&i)
+	if err := tx.Error; err != nil {
+		return nil, errors.NewInternalError(err.Error())
+	}
+	log.Printf("updated iteration to %v\n", i)
```

Comment:
thanks, fixed in https://github.com/almighty/almighty-core/pull/652/commits/bcc81828a5d13cc92cd49634172d8506296fd4ff